pkg/controllermanager/controller/controllerinstallation/controllerinstallation_control.go

```diff
@@ -229,26 +229,29 @@ func (c *defaultControllerInstallationControl) reconcile(controllerInstallation
 		return err
 	}
 
-	// Mix-in some standard values for seed.
 	seedValues := map[string]interface{}{
+		"identity":      seed.Name,
+		"provider":      seedCloudProvider,
+		"region":        seed.Spec.Cloud.Region,
+		"ingressDomain": seed.Spec.IngressDomain,
+		"blockCIDRs":    seed.Spec.BlockCIDRs,
+		"protected":     seed.Spec.Protected,
+		"visible":       seed.Spec.Visible,
+		"networks":      seed.Spec.Networks,
+	}
+
+	patchSeedValues(seedValues, seed)
```

Comment:
Imho, we can add `volumeProvider` directly to the list above (see previous comment). Extension controllers can decide themselves if they need to evaluate this field.